lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -335,6 +335,8 @@ def create_tar(cv, cvv, repositories, json)
 
             `tar cvfh #{repo_tar} #{repo_dirs.join(" ")}`
           end
+        else
+          puts "Ensure the content view '#{cvv['name']}' has atleast one repository."
```

Comment:
There's still a typo here: s/atleast/at least/

Reply:
Yes, will change this too.